src/services/earn/types.ts

```diff
@@ -13,6 +13,7 @@ export type IEarnService = {
   buildCreatePositionTx(_: CreateEarnPositionParams): Promise<BuiltTransaction>;
   buildIncreasePositionTx(_: IncreaseEarnPositionParams): Promise<BuiltTransaction>;
   buildWithdrawPositionTx(_: WithdrawEarnPositionParams): Promise<BuiltTransaction>;
+  getSupportedStrategies(_?: { chains?: ChainId[]; config?: { timeout: TimeString } }): Promise<Record<ChainId, Strategy[]>>;
```

Comment:
Just in case we want to add more config in the future
```suggestion
  getSupportedStrategies(_?: { chains?: ChainId[]; config?: { timeout?: TimeString } }): Promise<Record<ChainId, Strategy[]>>;
```